includes/REST/StoreSettingController.php

```diff
@@ -83,10 +121,49 @@ protected function get_vendor() {
         }
 
         if ( ! $current_user ) {
-            return new WP_Error( 'Unauthorized', __( 'You are not logged in' ), [ 'code' => 401 ] );
+            return new WP_Error( 'Unauthorized', __( 'You are not logged in', 'dokan-lite' ), [ 'code' => 401 ] );
         }
 
         return $vendor;
     }
 
+    /**
+     * Prepare links for the request.
+     *
+     * @param \WC_Data $object Object data.
+     * @param \WP_REST_Request $request Request object.
+     *
+     * @return array                   Links for the given post.
+     */
+    protected function prepare_links( $object, $request ) {
+        $links = [
+            'self'       => [
+                'href' => rest_url( sprintf( '/%s/%s/%d', $this->namespace, $this->rest_base, $object['id'] ) ),
+            ],
+            'collection' => [
+                'href' => rest_url( sprintf( '/%s/%s', $this->namespace, $this->rest_base ) ),
+            ],
+        ];
+
+        return $links;
+    }
+
+    /**
+     * Prepare a single user output for response
```

Comment:
Maybe consider changing the docblock title to **Prepare a single item output for response**